gnuradio-runtime/lib/block.cc

```diff
@@ -748,8 +748,11 @@ namespace gr {
         basic_block_sptr blk = global_block_registry.block_lookup(block);
         blk->post(port, pmt::cons(pmt::mp("done"), pmt::mp(true)));
 
-        //GR_LOG_DEBUG (d_debug_logger, "notify finished -->");
-        //pmt::print(pmt::cons(block,port));
+        std::ostringstream msg;
+        msg << "notify finished -->";
+        msg << pmt::cons(block,port);
+        GR_LOG_DEBUG (d_debug_logger, msg.str());
+        pmt::print(pmt::cons(block,port));
```

Comment:
This still prints to stdout. I think, it should `pmt::write_string` inside  `GR_LOG_DEBUG`